process_qsos.m

```diff
@@ -157,6 +163,12 @@
 
         min_observed_lambda = max(min_pos_lambda, min(this_wavelengths));
         lambda_observed = (max_observed_lambda - min_observed_lambda);
+        
+        %Find probability for out-of-range model
+        bw_likelihoods = pdf(bw_model, this_flux(this_wavelengths < min_observed_lambda));
+        bw_log_likelihood = sum(log(bw_likelihoods));
+        rw_likelihoods = pdf(rw_model, this_flux(this_wavelengths > max_observed_lambda));
+        rw_log_likelihood = sum(log(rw_likelihoods));
```

Comment:
Thinking maybe we need to apply `~this_pixel_mask` here since we applied the mask in the learned script.